### PR TITLE
added `default_gamma` and `default_gamma_r` arguments to rattle methods

### LINKED ISSUE #1589: Add `default_gamma` and `default_gamma_r` to rattle methods
### Description

Add `default_gamma` and `default_gamma_r` arguments to the constructors of:
* `hoomd.md.methods.rattle.Brownian`
* `hoomd.md.methods.rattle.Langevin`
* `hoomd.md.methods.rattle.OverdampedViscous`

### Proposed solution

method = hoomd.md.methods.rattle.Brownian(..., default_gamma=1.0, default_gamma_r=(1.0, 1.0, 1.0))

### Additional context

Include these arguments for consistency with the non-rattle methods:
https://hoomd-blue--1586.org.readthedocs.build/en/1586/module-md-methods.html#hoomd.md.methods.Brownian

### PR DESCRIPTION
**Pull request by melodyyzh**
## Description
`default_gamma` and `default_gamma_r` arguments do not exist in the following constructors:

`hoomd.md.methods.rattle.Brownian`
`hoomd.md.methods.rattle.Langevin`
`hoomd.md.methods.rattle.OverdampedViscous`

This PR added these arguments to the corresponding classes and doc strings. 

## Motivation and context

Resolves #1589 

## How has this been tested?

Appended the test case before `test_default_gamma` function in `test_methods.py` 
## Change log

```

```

## Checklist:

- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/CONTRIBUTING.rst).
- [x] I agree with the terms of the [**HOOMD-blue Contributor Agreement**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/ContributorAgreement.md).
- [x] My name is on the list of contributors (`sphinx-doc/credits.rst`) in the pull request source branch.

### CODE REVIEW

**Review by tommy-waltmann — APPROVED** [review]
Nice work on this!

**Review by shihkual — APPROVED** [review]
LGTM!

File: hoomd/md/methods/rattle.py [hunk 1]
```diff
@@ -267,6 +290,8 @@ def __init__(self,
                  filter,
                  kT,
                  manifold_constraint,
+                 default_gamma=1.0,
+                 default_gamma_r=(1.0, 1.0, 1.0),
                  tally_reservoir_energy=False,
                  tolerance=0.000001):
```
Comment (joaander) [inline hoomd/md/methods/rattle.py]:
```suggestion
                 tally_reservoir_energy=False,
                 tolerance=0.000001,
                 default_gamma=1.0,
                 default_gamma_r=(1.0, 1.0, 1.0),):
```

**Review by joaander — APPROVED** [review]
Thanks! One additional change is needed which I will apply and then merge.

**Comment by joaander** [comment]
pre-commit.ci autofix

**Pull request merged**